app/views/users/_form.html.erb

```diff
@@ -12,9 +12,9 @@
 <%= label_tag :email %><br>
 <%= text_field_tag :email, @user.email, placeholder: "Email" %><br>
 <%= label_tag :password %><br>
-<%= text_field_tag :password, @user.password, placeholder: "Password" %><br>
+<%= text_field_tag :password, nil, placeholder: "Password" %><br>
```

Comment:
Awesome, our passwords are working properly with BCrypt.

Reply:
Yes! It doesn't let a user save without confirmation pw matching.